lib/quota/QuotaManager.js

```diff
@@ -65,6 +56,27 @@ class QuotaManager {
     });
   }
 
+  getAllPlanIdsWithSameSKU(planName, serviceName, serviceCatalog) {
+    return Promise.try(() => {
+      const planManagerName = _.find(catalog.plans, ['name', planName]).manager.name;
+      const splittedPlanNameWithoutVersionInfo = _.drop(planName.split('-')); // assumption here is that service plan names are versioned, and the format is like <version>-{...}-<tshirt-size>
+      splittedPlanNameWithoutVersionInfo.unshift(''); // and skuName will be only -{...}-<tshirt-size>   
+      const skuName = _.join(splittedPlanNameWithoutVersionInfo, '-');
```

Comment:
Above is correct, nothing wrong but hard to read and decipher. Instead can you try something like the below
```
const skuName = _.chain('planName')
.split('-')
.filter((idx, i) => i !== 0)
.join('-')
.value()
```
chains are used for scenarios like above where you are performing a list of operations in sequence.
Alternatively you can just get strip everything after first delimiter '-' as thats what you are getting
```
const firstIdx = planName.indexOf('-');
const skuName = planName.substring(planName.indexOf('-', firstIdx));
```

Reply:
Adjusted the code accordingly, using the second method, as it is easier to read.